2020-05-07 14:46:22,412 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 7356 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 14:46:22,418 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 14:46:22,419 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 14:46:24,323 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 14:46:24,325 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 14:46:24,325 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 14:46:24,417 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 14:46:24,555 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 14:46:24,850 restartedMain INFO (DruidDataSource.java:994) - {dataSource-1} inited
2020-05-07 14:46:26,926 restartedMain INFO (PropertySourcedRequestMappingHandlerMapping.java:69) - Mapped URL path [/v2/api-docs] onto method [springfox.documentation.swagger2.web.Swagger2Controller#getDocumentation(String, HttpServletRequest)]
2020-05-07 14:46:27,284 restartedMain INFO (DocumentationPluginsBootstrapper.java:160) - Context refreshed
2020-05-07 14:46:27,301 restartedMain INFO (DocumentationPluginsBootstrapper.java:163) - Found 1 custom documentation plugin(s)
2020-05-07 14:46:27,352 restartedMain INFO (ApiListingReferenceScanner.java:41) - Scanning for api listing references
2020-05-07 14:46:27,525 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: getUserByNameUsingGET_1
2020-05-07 14:46:27,526 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: getUserByNameUsingGET_2
2020-05-07 14:46:27,527 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: insertUsingPOST_1
2020-05-07 14:46:27,548 restartedMain INFO (DirectJDKLog.java:173) - Starting ProtocolHandler ["http-nio-8080"]
2020-05-07 14:46:27,554 restartedMain INFO (DirectJDKLog.java:173) - Pausing ProtocolHandler ["http-nio-8080"]
2020-05-07 14:46:27,555 restartedMain INFO (DirectJDKLog.java:173) - Stopping service [Tomcat]
2020-05-07 14:46:27,559 restartedMain INFO (DirectJDKLog.java:173) - Stopping ProtocolHandler ["http-nio-8080"]
2020-05-07 14:46:27,559 restartedMain INFO (DirectJDKLog.java:173) - Destroying ProtocolHandler ["http-nio-8080"]
2020-05-07 14:46:27,567 restartedMain ERROR (LoggingFailureAnalysisReporter.java:40) - 

***************************
APPLICATION FAILED TO START
***************************

Description:

Web server failed to start. Port 8080 was already in use.

Action:

Identify and stop the process that's listening on port 8080 or configure this application to listen on another port.

2020-05-07 14:46:27,591 restartedMain INFO (DruidDataSource.java:2029) - {dataSource-1} closing ...
2020-05-07 14:46:27,596 restartedMain INFO (DruidDataSource.java:2101) - {dataSource-1} closed
2020-05-07 14:47:05,567 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 11544 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 14:47:05,570 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 14:47:05,570 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 14:47:06,992 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 14:47:06,993 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 14:47:06,993 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 14:47:07,076 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 14:47:07,195 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 14:47:07,401 restartedMain INFO (DruidDataSource.java:994) - {dataSource-1} inited
2020-05-07 14:47:08,632 restartedMain INFO (PropertySourcedRequestMappingHandlerMapping.java:69) - Mapped URL path [/v2/api-docs] onto method [springfox.documentation.swagger2.web.Swagger2Controller#getDocumentation(String, HttpServletRequest)]
2020-05-07 14:47:09,014 restartedMain INFO (DocumentationPluginsBootstrapper.java:160) - Context refreshed
2020-05-07 14:47:09,028 restartedMain INFO (DocumentationPluginsBootstrapper.java:163) - Found 1 custom documentation plugin(s)
2020-05-07 14:47:09,061 restartedMain INFO (ApiListingReferenceScanner.java:41) - Scanning for api listing references
2020-05-07 14:47:09,232 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: getUserByNameUsingGET_1
2020-05-07 14:47:09,233 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: getUserByNameUsingGET_2
2020-05-07 14:47:09,235 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: insertUsingPOST_1
2020-05-07 14:47:09,257 restartedMain INFO (DirectJDKLog.java:173) - Starting ProtocolHandler ["http-nio-8080"]
2020-05-07 14:47:09,278 restartedMain INFO (StartupInfoLogger.java:61) - Started ItsmAppliction in 4.204 seconds (JVM running for 4.973)
2020-05-07 14:49:05,074 http-nio-8080-exec-3 INFO (DirectJDKLog.java:173) - Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-05-07 14:49:05,188 http-nio-8080-exec-3 DEBUG (BaseJdbcLogger.java:143) - ==>  Preparing: INSERT INTO t_oper_log ( method, request_method, request_param, oper_type, title, response_json, oper_location, oper_ip, oper_url, oper_name, business_type, status ) VALUES ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ) 
2020-05-07 14:49:05,424 http-nio-8080-exec-3 DEBUG (BaseJdbcLogger.java:143) - ==> Parameters: POST(String), org.mof.cc.itsm.controller.QuestionController.save()(String), {"quetion": "青海系统无法登陆","questionDetail": "new某某用户登陆，提示异常，信息如下：Exception","askUser": "test"}(String), 1(Integer), 问题管理(String), {"status":"success"}(String), 内网(String), 127.0.0.1(String), /api/question/(String), admin(String), 1(String), 1(Integer)
2020-05-07 14:49:05,435 http-nio-8080-exec-3 DEBUG (BaseJdbcLogger.java:143) - <==    Updates: 1
2020-05-07 14:49:35,745 Thread-7 INFO (DruidDataSource.java:2029) - {dataSource-1} closing ...
2020-05-07 14:49:35,751 Thread-7 INFO (DruidDataSource.java:2101) - {dataSource-1} closed
2020-05-07 14:49:36,159 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 11544 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 14:49:36,160 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 14:49:36,160 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 14:49:36,584 restartedMain WARN (DirectJDKLog.java:173) - The MBean registry cannot be disabled because it has already been initialised
2020-05-07 14:49:36,669 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 14:49:36,670 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 14:49:36,670 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 14:49:36,691 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 14:49:36,710 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 14:49:36,762 restartedMain INFO (DruidDataSource.java:994) - {dataSource-2} inited
2020-05-07 14:49:37,012 restartedMain INFO (PropertySourcedRequestMappingHandlerMapping.java:69) - Mapped URL path [/v2/api-docs] onto method [springfox.documentation.swagger2.web.Swagger2Controller#getDocumentation(String, HttpServletRequest)]
2020-05-07 14:49:37,153 restartedMain INFO (DocumentationPluginsBootstrapper.java:160) - Context refreshed
2020-05-07 14:49:37,154 restartedMain INFO (DocumentationPluginsBootstrapper.java:163) - Found 1 custom documentation plugin(s)
2020-05-07 14:49:37,160 restartedMain INFO (ApiListingReferenceScanner.java:41) - Scanning for api listing references
2020-05-07 14:49:37,212 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: getUserByNameUsingGET_1
2020-05-07 14:49:37,213 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: getUserByNameUsingGET_2
2020-05-07 14:49:37,214 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: insertUsingPOST_1
2020-05-07 14:49:37,223 restartedMain INFO (DirectJDKLog.java:173) - Starting ProtocolHandler ["http-nio-8080"]
2020-05-07 14:49:37,230 restartedMain INFO (StartupInfoLogger.java:61) - Started ItsmAppliction in 1.128 seconds (JVM running for 152.925)
2020-05-07 16:33:11,886 http-nio-8080-exec-2 INFO (DirectJDKLog.java:173) - Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-05-07 16:33:12,126 http-nio-8080-exec-2 DEBUG (BaseJdbcLogger.java:143) - ==>  Preparing: INSERT INTO t_oper_log ( method, request_method, request_param, oper_type, title, response_json, oper_location, oper_ip, oper_url, oper_name, business_type, status ) VALUES ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ) 
2020-05-07 16:33:12,144 http-nio-8080-exec-2 DEBUG (BaseJdbcLogger.java:143) - ==> Parameters: POST(String), org.mof.cc.itsm.controller.QuestionController.save()(String), {"quetion": "青海系统无法登陆","questionDetail": "new某某用户登陆，提示异常，信息如下：Exception","askUser": "test"}(String), 1(Integer), 问题管理(String), {"status":"success"}(String), 内网(String), 127.0.0.1(String), /api/question/(String), admin(String), 1(String), 1(Integer)
2020-05-07 16:33:12,157 http-nio-8080-exec-2 DEBUG (BaseJdbcLogger.java:143) - <==    Updates: 1
2020-05-07 16:33:12,939 http-nio-8080-exec-3 DEBUG (BaseJdbcLogger.java:143) - ==>  Preparing: INSERT INTO t_oper_log ( method, request_method, request_param, oper_type, title, response_json, oper_location, oper_ip, oper_url, oper_name, business_type, status ) VALUES ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ) 
2020-05-07 16:33:12,941 http-nio-8080-exec-3 DEBUG (BaseJdbcLogger.java:143) - ==> Parameters: POST(String), org.mof.cc.itsm.controller.QuestionController.save()(String), {"quetion": "青海系统无法登陆","questionDetail": "new某某用户登陆，提示异常，信息如下：Exception","askUser": "test"}(String), 1(Integer), 问题管理(String), {"status":"success"}(String), 内网(String), 127.0.0.1(String), /api/question/(String), admin(String), 1(String), 1(Integer)
2020-05-07 16:33:12,943 http-nio-8080-exec-3 DEBUG (BaseJdbcLogger.java:143) - <==    Updates: 1
2020-05-07 16:33:13,673 http-nio-8080-exec-4 DEBUG (BaseJdbcLogger.java:143) - ==>  Preparing: INSERT INTO t_oper_log ( method, request_method, request_param, oper_type, title, response_json, oper_location, oper_ip, oper_url, oper_name, business_type, status ) VALUES ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ) 
2020-05-07 16:33:13,675 http-nio-8080-exec-4 DEBUG (BaseJdbcLogger.java:143) - ==> Parameters: POST(String), org.mof.cc.itsm.controller.QuestionController.save()(String), {"quetion": "青海系统无法登陆","questionDetail": "new某某用户登陆，提示异常，信息如下：Exception","askUser": "test"}(String), 1(Integer), 问题管理(String), {"status":"success"}(String), 内网(String), 127.0.0.1(String), /api/question/(String), admin(String), 1(String), 1(Integer)
2020-05-07 16:33:13,680 http-nio-8080-exec-4 DEBUG (BaseJdbcLogger.java:143) - <==    Updates: 1
2020-05-07 16:33:14,283 http-nio-8080-exec-5 DEBUG (BaseJdbcLogger.java:143) - ==>  Preparing: INSERT INTO t_oper_log ( method, request_method, request_param, oper_type, title, response_json, oper_location, oper_ip, oper_url, oper_name, business_type, status ) VALUES ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ) 
2020-05-07 16:33:14,287 http-nio-8080-exec-5 DEBUG (BaseJdbcLogger.java:143) - ==> Parameters: POST(String), org.mof.cc.itsm.controller.QuestionController.save()(String), {"quetion": "青海系统无法登陆","questionDetail": "new某某用户登陆，提示异常，信息如下：Exception","askUser": "test"}(String), 1(Integer), 问题管理(String), {"status":"success"}(String), 内网(String), 127.0.0.1(String), /api/question/(String), admin(String), 1(String), 1(Integer)
2020-05-07 16:33:14,292 http-nio-8080-exec-5 DEBUG (BaseJdbcLogger.java:143) - <==    Updates: 1
2020-05-07 16:34:16,564 http-nio-8080-exec-8 DEBUG (BaseJdbcLogger.java:143) - ==>  Preparing: INSERT INTO t_oper_log ( method, request_method, request_param, oper_type, title, response_json, oper_location, oper_ip, oper_url, oper_name, business_type, status ) VALUES ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ) 
2020-05-07 16:34:16,566 http-nio-8080-exec-8 DEBUG (BaseJdbcLogger.java:143) - ==> Parameters: POST(String), org.mof.cc.itsm.controller.QuestionController.save()(String), {"quetion": "青海系统无法登陆","questionDetail": "new某某用户登陆，提示异常，信息如下：Exception","askUser": "test"}(String), 1(Integer), 问题管理(String), {"status":"success"}(String), 内网(String), 127.0.0.1(String), /api/question/(String), admin(String), 1(String), 1(Integer)
2020-05-07 16:34:16,569 http-nio-8080-exec-8 DEBUG (BaseJdbcLogger.java:143) - <==    Updates: 1
2020-05-07 16:34:17,499 http-nio-8080-exec-9 DEBUG (BaseJdbcLogger.java:143) - ==>  Preparing: INSERT INTO t_oper_log ( method, request_method, request_param, oper_type, title, response_json, oper_location, oper_ip, oper_url, oper_name, business_type, status ) VALUES ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ) 
2020-05-07 16:34:17,501 http-nio-8080-exec-9 DEBUG (BaseJdbcLogger.java:143) - ==> Parameters: POST(String), org.mof.cc.itsm.controller.QuestionController.save()(String), {"quetion": "青海系统无法登陆","questionDetail": "new某某用户登陆，提示异常，信息如下：Exception","askUser": "test"}(String), 1(Integer), 问题管理(String), {"status":"success"}(String), 内网(String), 127.0.0.1(String), /api/question/(String), admin(String), 1(String), 1(Integer)
2020-05-07 16:34:17,503 http-nio-8080-exec-9 DEBUG (BaseJdbcLogger.java:143) - <==    Updates: 1
2020-05-07 16:34:18,234 http-nio-8080-exec-10 DEBUG (BaseJdbcLogger.java:143) - ==>  Preparing: INSERT INTO t_oper_log ( method, request_method, request_param, oper_type, title, response_json, oper_location, oper_ip, oper_url, oper_name, business_type, status ) VALUES ( ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ? ) 
2020-05-07 16:34:18,236 http-nio-8080-exec-10 DEBUG (BaseJdbcLogger.java:143) - ==> Parameters: POST(String), org.mof.cc.itsm.controller.QuestionController.save()(String), {"quetion": "青海系统无法登陆","questionDetail": "new某某用户登陆，提示异常，信息如下：Exception","askUser": "test"}(String), 1(Integer), 问题管理(String), {"status":"success"}(String), 内网(String), 127.0.0.1(String), /api/question/(String), admin(String), 1(String), 1(Integer)
2020-05-07 16:34:18,244 http-nio-8080-exec-10 DEBUG (BaseJdbcLogger.java:143) - <==    Updates: 1
2020-05-07 16:35:40,863 Thread-15 INFO (DruidDataSource.java:2029) - {dataSource-2} closing ...
2020-05-07 16:35:40,871 Thread-15 INFO (DruidDataSource.java:2101) - {dataSource-2} closed
2020-05-07 16:35:41,453 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 11544 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 16:35:41,453 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 16:35:41,453 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 16:35:42,146 restartedMain WARN (DirectJDKLog.java:173) - The MBean registry cannot be disabled because it has already been initialised
2020-05-07 16:35:42,326 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 16:35:42,349 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 16:35:42,350 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 16:35:42,381 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 16:35:42,414 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 16:35:42,527 restartedMain INFO (DruidDataSource.java:994) - {dataSource-3} inited
2020-05-07 16:35:43,072 restartedMain INFO (PropertySourcedRequestMappingHandlerMapping.java:69) - Mapped URL path [/v2/api-docs] onto method [springfox.documentation.swagger2.web.Swagger2Controller#getDocumentation(String, HttpServletRequest)]
2020-05-07 16:35:43,303 restartedMain INFO (DocumentationPluginsBootstrapper.java:160) - Context refreshed
2020-05-07 16:35:43,304 restartedMain INFO (DocumentationPluginsBootstrapper.java:163) - Found 1 custom documentation plugin(s)
2020-05-07 16:35:43,312 restartedMain INFO (ApiListingReferenceScanner.java:41) - Scanning for api listing references
2020-05-07 16:35:43,395 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: getUserByNameUsingGET_1
2020-05-07 16:35:43,397 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: getUserByNameUsingGET_2
2020-05-07 16:35:43,399 restartedMain INFO (CachingOperationNameGenerator.java:40) - Generating unique operation named: insertUsingPOST_1
2020-05-07 16:35:43,409 restartedMain INFO (DirectJDKLog.java:173) - Starting ProtocolHandler ["http-nio-8080"]
2020-05-07 16:35:43,419 restartedMain INFO (StartupInfoLogger.java:61) - Started ItsmAppliction in 2.035 seconds (JVM running for 6519.114)
2020-05-07 16:35:48,745 RMI TCP Connection(114)-127.0.0.1 INFO (DruidDataSource.java:2029) - {dataSource-3} closing ...
2020-05-07 16:35:48,751 RMI TCP Connection(114)-127.0.0.1 INFO (DruidDataSource.java:2101) - {dataSource-3} closed
2020-05-07 17:59:49,032 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 13048 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 17:59:49,034 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 17:59:49,035 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 17:59:50,805 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 17:59:50,806 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 17:59:50,806 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 17:59:50,905 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 17:59:51,018 restartedMain INFO (QuartzInitializerListener.java:147) - Quartz Initializer Servlet loaded, initializing Scheduler...
2020-05-07 17:59:51,051 restartedMain INFO (StdSchedulerFactory.java:1024) - Using ConnectionProvider class 'org.quartz.utils.C3p0PoolingConnectionProvider' for data source 'qzDS'
2020-05-07 17:59:51,053 restartedMain ERROR (QuartzInitializerListener.java:232) - Quartz Scheduler failed to initialize: org.quartz.SchedulerException: Could not initialize DataSource: qzDS [See nested exception: org.quartz.SchedulerException: ConnectionProvider class 'org.quartz.utils.C3p0PoolingConnectionProvider' could not be instantiated. [See nested exception: java.lang.reflect.InvocationTargetException]]
2020-05-07 17:59:51,079 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 17:59:51,320 restartedMain INFO (DruidDataSource.java:994) - {dataSource-1} inited
2020-05-07 17:59:52,576 restartedMain WARN (AbstractApplicationContext.java:558) - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'jobAndTriggerService'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobAndTriggerServiceImpl': Unsatisfied dependency expressed through field 'jobAndTriggerMapper'; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.mof.cc.itsm.dao.JobAndTriggerMapper' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
2020-05-07 17:59:52,578 restartedMain INFO (DruidDataSource.java:2029) - {dataSource-1} closing ...
2020-05-07 17:59:52,584 restartedMain INFO (DruidDataSource.java:2101) - {dataSource-1} closed
2020-05-07 17:59:52,587 restartedMain INFO (DirectJDKLog.java:173) - Stopping service [Tomcat]
2020-05-07 17:59:52,589 restartedMain INFO (QuartzInitializerListener.java:264) - Quartz Scheduler successful shutdown.
2020-05-07 17:59:52,700 restartedMain ERROR (LoggingFailureAnalysisReporter.java:40) - 

***************************
APPLICATION FAILED TO START
***************************

Description:

Field jobAndTriggerMapper in org.mof.cc.itsm.service.impl.JobAndTriggerServiceImpl required a bean of type 'org.mof.cc.itsm.dao.JobAndTriggerMapper' that could not be found.

The injection point has the following annotations:
	- @org.springframework.beans.factory.annotation.Autowired(required=true)


Action:

Consider defining a bean of type 'org.mof.cc.itsm.dao.JobAndTriggerMapper' in your configuration.

2020-05-07 18:02:37,999 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 18476 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 18:02:38,002 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 18:02:38,003 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 18:02:39,508 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 18:02:39,509 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 18:02:39,509 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 18:02:39,596 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 18:02:39,645 restartedMain INFO (QuartzInitializerListener.java:147) - Quartz Initializer Servlet loaded, initializing Scheduler...
2020-05-07 18:02:39,763 restartedMain INFO (StdSchedulerFactory.java:1024) - Using ConnectionProvider class 'org.quartz.utils.C3p0PoolingConnectionProvider' for data source 'qzDS'
2020-05-07 18:02:39,790 MLog-Init-Reporter INFO (Slf4jMLog.java:212) - MLog clients using slf4j logging.
2020-05-07 18:02:39,832 restartedMain INFO (Slf4jMLog.java:212) - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2020-05-07 18:02:39,885 restartedMain INFO (StdSchedulerFactory.java:1208) - Using default implementation for ThreadExecutor
2020-05-07 18:02:39,887 restartedMain INFO (SimpleThreadPool.java:268) - Job execution threads will use class loader of thread: restartedMain
2020-05-07 18:02:39,896 restartedMain INFO (SchedulerSignalerImpl.java:61) - Initialized Scheduler Signaller of type: class org.quartz.core.SchedulerSignalerImpl
2020-05-07 18:02:39,896 restartedMain INFO (QuartzScheduler.java:229) - Quartz Scheduler v.2.3.1 created.
2020-05-07 18:02:39,897 restartedMain INFO (JobStoreSupport.java:675) - Using thread monitor-based data access locking (synchronization).
2020-05-07 18:02:39,898 restartedMain INFO (JobStoreTX.java:59) - JobStoreTX initialized.
2020-05-07 18:02:39,899 restartedMain INFO (QuartzScheduler.java:294) - Scheduler meta-data: Quartz Scheduler (v2.3.1) 'DefaultQuartzScheduler' with instanceId 'NON_CLUSTERED'
  Scheduler class: 'org.quartz.core.QuartzScheduler' - running locally.
  NOT STARTED.
  Currently in standby mode.
  Number of jobs executed: 0
  Using thread pool 'org.quartz.simpl.SimpleThreadPool' - with 5 threads.
  Using job-store 'org.quartz.impl.jdbcjobstore.JobStoreTX' - which supports persistence. and is not clustered.

2020-05-07 18:02:39,899 restartedMain INFO (StdSchedulerFactory.java:1362) - Quartz scheduler 'DefaultQuartzScheduler' initialized from default resource file in Quartz package: 'quartz.properties'
2020-05-07 18:02:39,899 restartedMain INFO (StdSchedulerFactory.java:1366) - Quartz scheduler version: 2.3.1
2020-05-07 18:02:39,925 restartedMain INFO (Slf4jMLog.java:212) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 2zmh4sa91u76hjfntahpf|7b0a4e0a, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 2zmh4sa91u76hjfntahpf|7b0a4e0a, idleConnectionTestPeriod -> 0, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://127.0.0.1:3306/callcenter?useUnicode=true&characterEncoding=UTF-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 10, maxStatements -> 0, maxStatementsPerConnection -> 120, minPoolSize -> 1, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
2020-05-07 18:03:10,234 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u76hjfntahpf|7b0a4e0a]-HelperThread-#1 WARN (Slf4jMLog.java:223) - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@242216df -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
java.sql.SQLException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:76)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:836)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1176)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1163)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1908)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at sun.reflect.GeneratedConstructorAccessor38.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.util.TimeUtil.getCanonicalTimezone(TimeUtil.java:132)
	at com.mysql.cj.protocol.a.NativeProtocol.configureTimezone(NativeProtocol.java:2121)
	at com.mysql.cj.protocol.a.NativeProtocol.initServerSession(NativeProtocol.java:2145)
	at com.mysql.cj.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:1310)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:967)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:826)
	... 12 common frames omitted
2020-05-07 18:03:10,234 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u76hjfntahpf|7b0a4e0a]-HelperThread-#1 WARN (Slf4jMLog.java:220) - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@78efe140 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
2020-05-07 18:03:10,239 restartedMain ERROR (QuartzInitializerListener.java:232) - Quartz Scheduler failed to initialize: org.quartz.SchedulerConfigException: Failure occured during job recovery. [See nested exception: org.quartz.JobPersistenceException: Failed to obtain DB connection from data source 'qzDS': java.sql.SQLException: Connections could not be acquired from the underlying database! [See nested exception: java.sql.SQLException: Connections could not be acquired from the underlying database!]]
2020-05-07 18:03:10,243 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u76hjfntahpf|7b0a4e0a]-HelperThread-#1 WARN (Slf4jMLog.java:223) - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@40da3b92 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
java.sql.SQLException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:76)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:836)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1176)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1163)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1908)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at sun.reflect.GeneratedConstructorAccessor38.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.util.TimeUtil.getCanonicalTimezone(TimeUtil.java:132)
	at com.mysql.cj.protocol.a.NativeProtocol.configureTimezone(NativeProtocol.java:2121)
	at com.mysql.cj.protocol.a.NativeProtocol.initServerSession(NativeProtocol.java:2145)
	at com.mysql.cj.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:1310)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:967)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:826)
	... 12 common frames omitted
2020-05-07 18:03:10,243 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u76hjfntahpf|7b0a4e0a]-HelperThread-#1 WARN (Slf4jMLog.java:220) - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@78efe140 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
2020-05-07 18:03:10,250 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u76hjfntahpf|7b0a4e0a]-HelperThread-#1 WARN (Slf4jMLog.java:223) - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@2cff8b41 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
java.sql.SQLException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:76)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:836)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1176)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1163)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1908)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at sun.reflect.GeneratedConstructorAccessor38.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.util.TimeUtil.getCanonicalTimezone(TimeUtil.java:132)
	at com.mysql.cj.protocol.a.NativeProtocol.configureTimezone(NativeProtocol.java:2121)
	at com.mysql.cj.protocol.a.NativeProtocol.initServerSession(NativeProtocol.java:2145)
	at com.mysql.cj.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:1310)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:967)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:826)
	... 12 common frames omitted
2020-05-07 18:03:10,250 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u76hjfntahpf|7b0a4e0a]-HelperThread-#1 WARN (Slf4jMLog.java:220) - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@78efe140 is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
2020-05-07 18:03:10,273 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 18:03:10,410 restartedMain INFO (DruidDataSource.java:994) - {dataSource-1} inited
2020-05-07 18:03:11,430 restartedMain WARN (AbstractApplicationContext.java:558) - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'jobAndTriggerService'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobAndTriggerServiceImpl': Unsatisfied dependency expressed through field 'jobAndTriggerMapper'; nested exception is org.springframework.beans.factory.NoSuchBeanDefinitionException: No qualifying bean of type 'org.mof.cc.itsm.dao.JobAndTriggerMapper' available: expected at least 1 bean which qualifies as autowire candidate. Dependency annotations: {@org.springframework.beans.factory.annotation.Autowired(required=true)}
2020-05-07 18:03:11,432 restartedMain INFO (DruidDataSource.java:2029) - {dataSource-1} closing ...
2020-05-07 18:03:11,436 restartedMain INFO (DruidDataSource.java:2101) - {dataSource-1} closed
2020-05-07 18:03:11,439 restartedMain INFO (DirectJDKLog.java:173) - Stopping service [Tomcat]
2020-05-07 18:03:11,441 restartedMain INFO (QuartzScheduler.java:666) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutting down.
2020-05-07 18:03:11,441 restartedMain INFO (QuartzScheduler.java:585) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED paused.
2020-05-07 18:03:11,442 restartedMain INFO (QuartzScheduler.java:740) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutdown complete.
2020-05-07 18:03:11,442 restartedMain INFO (QuartzInitializerListener.java:264) - Quartz Scheduler successful shutdown.
2020-05-07 18:03:11,444 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-1] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:03:11,445 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-2] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:03:11,445 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-3] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:03:11,446 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-4] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:03:11,446 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-5] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:03:11,532 restartedMain ERROR (LoggingFailureAnalysisReporter.java:40) - 

***************************
APPLICATION FAILED TO START
***************************

Description:

Field jobAndTriggerMapper in org.mof.cc.itsm.service.impl.JobAndTriggerServiceImpl required a bean of type 'org.mof.cc.itsm.dao.JobAndTriggerMapper' that could not be found.

The injection point has the following annotations:
	- @org.springframework.beans.factory.annotation.Autowired(required=true)


Action:

Consider defining a bean of type 'org.mof.cc.itsm.dao.JobAndTriggerMapper' in your configuration.

2020-05-07 18:08:29,423 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 23536 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 18:08:29,426 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 18:08:29,426 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 18:08:30,839 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 18:08:30,839 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 18:08:30,840 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 18:08:30,927 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 18:08:31,030 restartedMain INFO (QuartzInitializerListener.java:147) - Quartz Initializer Servlet loaded, initializing Scheduler...
2020-05-07 18:08:31,058 restartedMain INFO (StdSchedulerFactory.java:1024) - Using ConnectionProvider class 'org.quartz.utils.C3p0PoolingConnectionProvider' for data source 'qzDS'
2020-05-07 18:08:31,073 MLog-Init-Reporter INFO (Slf4jMLog.java:212) - MLog clients using slf4j logging.
2020-05-07 18:08:31,104 restartedMain INFO (Slf4jMLog.java:212) - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2020-05-07 18:08:31,150 restartedMain INFO (StdSchedulerFactory.java:1208) - Using default implementation for ThreadExecutor
2020-05-07 18:08:31,152 restartedMain INFO (SimpleThreadPool.java:268) - Job execution threads will use class loader of thread: restartedMain
2020-05-07 18:08:31,160 restartedMain INFO (SchedulerSignalerImpl.java:61) - Initialized Scheduler Signaller of type: class org.quartz.core.SchedulerSignalerImpl
2020-05-07 18:08:31,160 restartedMain INFO (QuartzScheduler.java:229) - Quartz Scheduler v.2.3.1 created.
2020-05-07 18:08:31,161 restartedMain INFO (JobStoreSupport.java:675) - Using thread monitor-based data access locking (synchronization).
2020-05-07 18:08:31,162 restartedMain INFO (JobStoreTX.java:59) - JobStoreTX initialized.
2020-05-07 18:08:31,162 restartedMain INFO (QuartzScheduler.java:294) - Scheduler meta-data: Quartz Scheduler (v2.3.1) 'DefaultQuartzScheduler' with instanceId 'NON_CLUSTERED'
  Scheduler class: 'org.quartz.core.QuartzScheduler' - running locally.
  NOT STARTED.
  Currently in standby mode.
  Number of jobs executed: 0
  Using thread pool 'org.quartz.simpl.SimpleThreadPool' - with 5 threads.
  Using job-store 'org.quartz.impl.jdbcjobstore.JobStoreTX' - which supports persistence. and is not clustered.

2020-05-07 18:08:31,163 restartedMain INFO (StdSchedulerFactory.java:1362) - Quartz scheduler 'DefaultQuartzScheduler' initialized from default resource file in Quartz package: 'quartz.properties'
2020-05-07 18:08:31,163 restartedMain INFO (StdSchedulerFactory.java:1366) - Quartz scheduler version: 2.3.1
2020-05-07 18:08:31,183 restartedMain INFO (Slf4jMLog.java:212) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 2zmh4sa91u7e0l272jaha|4a8b98e9, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 2zmh4sa91u7e0l272jaha|4a8b98e9, idleConnectionTestPeriod -> 0, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://127.0.0.1:3306/callcenter?useUnicode=true&characterEncoding=UTF-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 10, maxStatements -> 0, maxStatementsPerConnection -> 120, minPoolSize -> 1, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
2020-05-07 18:09:01,556 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7e0l272jaha|4a8b98e9]-HelperThread-#2 WARN (Slf4jMLog.java:223) - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@51e2c0b0 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
java.sql.SQLException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:76)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:836)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1176)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1163)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1908)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at sun.reflect.GeneratedConstructorAccessor38.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.util.TimeUtil.getCanonicalTimezone(TimeUtil.java:132)
	at com.mysql.cj.protocol.a.NativeProtocol.configureTimezone(NativeProtocol.java:2121)
	at com.mysql.cj.protocol.a.NativeProtocol.initServerSession(NativeProtocol.java:2145)
	at com.mysql.cj.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:1310)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:967)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:826)
	... 12 common frames omitted
2020-05-07 18:09:01,556 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7e0l272jaha|4a8b98e9]-HelperThread-#1 WARN (Slf4jMLog.java:223) - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@1598063a -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
java.sql.SQLException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:76)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:836)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1176)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1163)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1908)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at sun.reflect.GeneratedConstructorAccessor38.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.util.TimeUtil.getCanonicalTimezone(TimeUtil.java:132)
	at com.mysql.cj.protocol.a.NativeProtocol.configureTimezone(NativeProtocol.java:2121)
	at com.mysql.cj.protocol.a.NativeProtocol.initServerSession(NativeProtocol.java:2145)
	at com.mysql.cj.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:1310)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:967)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:826)
	... 12 common frames omitted
2020-05-07 18:09:01,557 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7e0l272jaha|4a8b98e9]-HelperThread-#2 WARN (Slf4jMLog.java:220) - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@5c20764d is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
2020-05-07 18:09:01,559 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7e0l272jaha|4a8b98e9]-HelperThread-#1 WARN (Slf4jMLog.java:220) - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@5c20764d is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
2020-05-07 18:09:01,560 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7e0l272jaha|4a8b98e9]-HelperThread-#0 WARN (Slf4jMLog.java:223) - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@2d126edc -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
java.sql.SQLException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:76)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:836)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1176)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1163)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1908)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at sun.reflect.GeneratedConstructorAccessor38.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.util.TimeUtil.getCanonicalTimezone(TimeUtil.java:132)
	at com.mysql.cj.protocol.a.NativeProtocol.configureTimezone(NativeProtocol.java:2121)
	at com.mysql.cj.protocol.a.NativeProtocol.initServerSession(NativeProtocol.java:2145)
	at com.mysql.cj.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:1310)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:967)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:826)
	... 12 common frames omitted
2020-05-07 18:09:01,560 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7e0l272jaha|4a8b98e9]-HelperThread-#0 WARN (Slf4jMLog.java:220) - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@5c20764d is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
2020-05-07 18:09:01,560 restartedMain ERROR (QuartzInitializerListener.java:232) - Quartz Scheduler failed to initialize: org.quartz.SchedulerConfigException: Failure occured during job recovery. [See nested exception: org.quartz.JobPersistenceException: Failed to obtain DB connection from data source 'qzDS': java.sql.SQLException: Connections could not be acquired from the underlying database! [See nested exception: java.sql.SQLException: Connections could not be acquired from the underlying database!]]
2020-05-07 18:09:01,586 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 18:09:01,731 restartedMain INFO (DruidDataSource.java:994) - {dataSource-1} inited
2020-05-07 18:09:02,712 restartedMain WARN (AbstractApplicationContext.java:558) - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'scheduler'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
2020-05-07 18:09:02,715 restartedMain INFO (DruidDataSource.java:2029) - {dataSource-1} closing ...
2020-05-07 18:09:02,719 restartedMain INFO (DruidDataSource.java:2101) - {dataSource-1} closed
2020-05-07 18:09:02,721 restartedMain INFO (DirectJDKLog.java:173) - Stopping service [Tomcat]
2020-05-07 18:09:02,723 restartedMain INFO (QuartzScheduler.java:666) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutting down.
2020-05-07 18:09:02,723 restartedMain INFO (QuartzScheduler.java:585) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED paused.
2020-05-07 18:09:02,725 restartedMain INFO (QuartzScheduler.java:740) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutdown complete.
2020-05-07 18:09:02,725 restartedMain INFO (QuartzInitializerListener.java:264) - Quartz Scheduler successful shutdown.
2020-05-07 18:09:02,727 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-1] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:09:02,728 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-2] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:09:02,729 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-3] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:09:02,729 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-4] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:09:02,730 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-5] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:09:02,742 restartedMain ERROR (SpringApplication.java:826) - Application run failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'scheduler'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:639)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:116)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:397)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1429)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:879)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:141)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.mof.cc.itsm.ItsmAppliction.main(ItsmAppliction.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.devtools.restart.RestartLauncher.run(RestartLauncher.java:49)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:645)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:475)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1338)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1287)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1207)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:636)
	... 24 common frames omitted
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:640)
	... 37 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1803)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:595)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:339)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$743d7ed1.schedulerFactoryBean(<generated>)
	at org.mof.cc.itsm.config.SchedulerConfig.scheduler(SchedulerConfig.java:54)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$743d7ed1.CGLIB$scheduler$0(<generated>)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$743d7ed1$$FastClassBySpringCGLIB$$a7087802.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:363)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$743d7ed1.scheduler(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	... 38 common frames omitted
Caused by: java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.createScheduler(SchedulerFactoryBean.java:679)
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.prepareScheduler(SchedulerFactoryBean.java:614)
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.afterPropertiesSet(SchedulerFactoryBean.java:502)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1862)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1799)
	... 57 common frames omitted
2020-05-07 18:11:21,475 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 3436 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 18:11:21,477 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 18:11:21,478 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 18:11:22,818 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 18:11:22,819 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 18:11:22,819 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 18:11:22,904 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 18:11:23,009 restartedMain INFO (QuartzInitializerListener.java:147) - Quartz Initializer Servlet loaded, initializing Scheduler...
2020-05-07 18:11:23,036 restartedMain INFO (StdSchedulerFactory.java:1024) - Using ConnectionProvider class 'org.quartz.utils.C3p0PoolingConnectionProvider' for data source 'qzDS'
2020-05-07 18:11:23,052 MLog-Init-Reporter INFO (Slf4jMLog.java:212) - MLog clients using slf4j logging.
2020-05-07 18:11:23,085 restartedMain INFO (Slf4jMLog.java:212) - Initializing c3p0-0.9.5.5 [built 11-December-2019 22:18:33 -0800; debug? true; trace: 10]
2020-05-07 18:11:23,131 restartedMain INFO (StdSchedulerFactory.java:1208) - Using default implementation for ThreadExecutor
2020-05-07 18:11:23,132 restartedMain INFO (SimpleThreadPool.java:268) - Job execution threads will use class loader of thread: restartedMain
2020-05-07 18:11:23,141 restartedMain INFO (SchedulerSignalerImpl.java:61) - Initialized Scheduler Signaller of type: class org.quartz.core.SchedulerSignalerImpl
2020-05-07 18:11:23,141 restartedMain INFO (QuartzScheduler.java:229) - Quartz Scheduler v.2.3.1 created.
2020-05-07 18:11:23,142 restartedMain INFO (JobStoreSupport.java:675) - Using thread monitor-based data access locking (synchronization).
2020-05-07 18:11:23,143 restartedMain INFO (JobStoreTX.java:59) - JobStoreTX initialized.
2020-05-07 18:11:23,144 restartedMain INFO (QuartzScheduler.java:294) - Scheduler meta-data: Quartz Scheduler (v2.3.1) 'DefaultQuartzScheduler' with instanceId 'NON_CLUSTERED'
  Scheduler class: 'org.quartz.core.QuartzScheduler' - running locally.
  NOT STARTED.
  Currently in standby mode.
  Number of jobs executed: 0
  Using thread pool 'org.quartz.simpl.SimpleThreadPool' - with 5 threads.
  Using job-store 'org.quartz.impl.jdbcjobstore.JobStoreTX' - which supports persistence. and is not clustered.

2020-05-07 18:11:23,144 restartedMain INFO (StdSchedulerFactory.java:1362) - Quartz scheduler 'DefaultQuartzScheduler' initialized from default resource file in Quartz package: 'quartz.properties'
2020-05-07 18:11:23,144 restartedMain INFO (StdSchedulerFactory.java:1366) - Quartz scheduler version: 2.3.1
2020-05-07 18:11:23,167 restartedMain INFO (Slf4jMLog.java:212) - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 2zmh4sa91u7hpa81bygtal|4af009cb, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> com.mysql.jdbc.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 2zmh4sa91u7hpa81bygtal|4af009cb, idleConnectionTestPeriod -> 0, initialPoolSize -> 3, jdbcUrl -> jdbc:mysql://127.0.0.1:3306/callcenter?useUnicode=true&characterEncoding=UTF-8, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 10, maxStatements -> 0, maxStatementsPerConnection -> 120, minPoolSize -> 1, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
2020-05-07 18:11:53,502 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7hpa81bygtal|4af009cb]-HelperThread-#2 WARN (Slf4jMLog.java:223) - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@d4cfdae -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
java.sql.SQLException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:76)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:836)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1176)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1163)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1908)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at sun.reflect.GeneratedConstructorAccessor39.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.util.TimeUtil.getCanonicalTimezone(TimeUtil.java:132)
	at com.mysql.cj.protocol.a.NativeProtocol.configureTimezone(NativeProtocol.java:2121)
	at com.mysql.cj.protocol.a.NativeProtocol.initServerSession(NativeProtocol.java:2145)
	at com.mysql.cj.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:1310)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:967)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:826)
	... 12 common frames omitted
2020-05-07 18:11:53,503 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7hpa81bygtal|4af009cb]-HelperThread-#2 WARN (Slf4jMLog.java:220) - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@2bb21a6b is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
2020-05-07 18:11:53,506 restartedMain ERROR (QuartzInitializerListener.java:232) - Quartz Scheduler failed to initialize: org.quartz.SchedulerConfigException: Failure occured during job recovery. [See nested exception: org.quartz.JobPersistenceException: Failed to obtain DB connection from data source 'qzDS': java.sql.SQLException: Connections could not be acquired from the underlying database! [See nested exception: java.sql.SQLException: Connections could not be acquired from the underlying database!]]
2020-05-07 18:11:53,515 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7hpa81bygtal|4af009cb]-HelperThread-#2 WARN (Slf4jMLog.java:223) - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@515b9987 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
java.sql.SQLException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:76)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:836)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1176)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1163)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1908)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at sun.reflect.GeneratedConstructorAccessor39.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.util.TimeUtil.getCanonicalTimezone(TimeUtil.java:132)
	at com.mysql.cj.protocol.a.NativeProtocol.configureTimezone(NativeProtocol.java:2121)
	at com.mysql.cj.protocol.a.NativeProtocol.initServerSession(NativeProtocol.java:2145)
	at com.mysql.cj.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:1310)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:967)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:826)
	... 12 common frames omitted
2020-05-07 18:11:53,515 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7hpa81bygtal|4af009cb]-HelperThread-#2 WARN (Slf4jMLog.java:220) - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@2bb21a6b is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
2020-05-07 18:11:53,522 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7hpa81bygtal|4af009cb]-HelperThread-#2 WARN (Slf4jMLog.java:223) - com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@6f574d61 -- Acquisition Attempt Failed!!! Clearing pending acquires. While trying to acquire a needed new resource, we failed to succeed more than the maximum number of allowed acquisition attempts (30). Last acquisition attempt exception: 
java.sql.SQLException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:97)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:89)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:63)
	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:73)
	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:76)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:836)
	at com.mysql.cj.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:456)
	at com.mysql.cj.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:246)
	at com.mysql.cj.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:199)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1176)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1163)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1908)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
Caused by: com.mysql.cj.exceptions.InvalidConnectionAttributeException: The server time zone value '�й���׼ʱ��' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.
	at sun.reflect.GeneratedConstructorAccessor39.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:61)
	at com.mysql.cj.exceptions.ExceptionFactory.createException(ExceptionFactory.java:85)
	at com.mysql.cj.util.TimeUtil.getCanonicalTimezone(TimeUtil.java:132)
	at com.mysql.cj.protocol.a.NativeProtocol.configureTimezone(NativeProtocol.java:2121)
	at com.mysql.cj.protocol.a.NativeProtocol.initServerSession(NativeProtocol.java:2145)
	at com.mysql.cj.jdbc.ConnectionImpl.initializePropsFromServer(ConnectionImpl.java:1310)
	at com.mysql.cj.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:967)
	at com.mysql.cj.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:826)
	... 12 common frames omitted
2020-05-07 18:11:53,522 C3P0PooledConnectionPoolManager[identityToken->2zmh4sa91u7hpa81bygtal|4af009cb]-HelperThread-#2 WARN (Slf4jMLog.java:220) - Having failed to acquire a resource, com.mchange.v2.resourcepool.BasicResourcePool@2bb21a6b is interrupting all Threads waiting on a resource to check out. Will try again in response to new client requests.
2020-05-07 18:11:53,534 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 18:11:53,656 restartedMain INFO (DruidDataSource.java:994) - {dataSource-1} inited
2020-05-07 18:11:54,556 restartedMain WARN (AbstractApplicationContext.java:558) - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'scheduler'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
2020-05-07 18:11:54,560 restartedMain INFO (DruidDataSource.java:2029) - {dataSource-1} closing ...
2020-05-07 18:11:54,565 restartedMain INFO (DruidDataSource.java:2101) - {dataSource-1} closed
2020-05-07 18:11:54,567 restartedMain INFO (DirectJDKLog.java:173) - Stopping service [Tomcat]
2020-05-07 18:11:54,569 restartedMain INFO (QuartzScheduler.java:666) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutting down.
2020-05-07 18:11:54,569 restartedMain INFO (QuartzScheduler.java:585) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED paused.
2020-05-07 18:11:54,571 restartedMain INFO (QuartzScheduler.java:740) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutdown complete.
2020-05-07 18:11:54,571 restartedMain INFO (QuartzInitializerListener.java:264) - Quartz Scheduler successful shutdown.
2020-05-07 18:11:54,573 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-1] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:11:54,574 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-2] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:11:54,575 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-3] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:11:54,575 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-4] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:11:54,576 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-5] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:11:54,589 restartedMain ERROR (SpringApplication.java:826) - Application run failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'scheduler'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:639)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:116)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:397)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1429)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:879)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:141)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.mof.cc.itsm.ItsmAppliction.main(ItsmAppliction.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.devtools.restart.RestartLauncher.run(RestartLauncher.java:49)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:645)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:475)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1338)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1287)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1207)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:636)
	... 24 common frames omitted
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:640)
	... 37 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1803)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:595)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:339)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$182cc793.schedulerFactoryBean(<generated>)
	at org.mof.cc.itsm.config.SchedulerConfig.scheduler(SchedulerConfig.java:54)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$182cc793.CGLIB$scheduler$0(<generated>)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$182cc793$$FastClassBySpringCGLIB$$9aab61f.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:363)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$182cc793.scheduler(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	... 38 common frames omitted
Caused by: java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.createScheduler(SchedulerFactoryBean.java:679)
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.prepareScheduler(SchedulerFactoryBean.java:614)
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.afterPropertiesSet(SchedulerFactoryBean.java:502)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1862)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1799)
	... 57 common frames omitted
2020-05-07 18:15:38,561 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 21384 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 18:15:38,563 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 18:15:38,564 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 18:15:40,034 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 18:15:40,035 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 18:15:40,035 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 18:15:40,123 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 18:15:40,236 restartedMain INFO (QuartzInitializerListener.java:147) - Quartz Initializer Servlet loaded, initializing Scheduler...
2020-05-07 18:15:40,267 restartedMain INFO (StdSchedulerFactory.java:1208) - Using default implementation for ThreadExecutor
2020-05-07 18:15:40,268 restartedMain INFO (SimpleThreadPool.java:268) - Job execution threads will use class loader of thread: restartedMain
2020-05-07 18:15:40,277 restartedMain INFO (SchedulerSignalerImpl.java:61) - Initialized Scheduler Signaller of type: class org.quartz.core.SchedulerSignalerImpl
2020-05-07 18:15:40,277 restartedMain INFO (QuartzScheduler.java:229) - Quartz Scheduler v.2.3.1 created.
2020-05-07 18:15:40,278 restartedMain INFO (RAMJobStore.java:155) - RAMJobStore initialized.
2020-05-07 18:15:40,279 restartedMain INFO (QuartzScheduler.java:294) - Scheduler meta-data: Quartz Scheduler (v2.3.1) 'DefaultQuartzScheduler' with instanceId 'NON_CLUSTERED'
  Scheduler class: 'org.quartz.core.QuartzScheduler' - running locally.
  NOT STARTED.
  Currently in standby mode.
  Number of jobs executed: 0
  Using thread pool 'org.quartz.simpl.SimpleThreadPool' - with 10 threads.
  Using job-store 'org.quartz.simpl.RAMJobStore' - which does not support persistence. and is not clustered.

2020-05-07 18:15:40,279 restartedMain INFO (StdSchedulerFactory.java:1362) - Quartz scheduler 'DefaultQuartzScheduler' initialized from default resource file in Quartz package: 'quartz.properties'
2020-05-07 18:15:40,279 restartedMain INFO (StdSchedulerFactory.java:1366) - Quartz scheduler version: 2.3.1
2020-05-07 18:15:40,279 restartedMain INFO (QuartzScheduler.java:547) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED started.
2020-05-07 18:15:40,280 restartedMain INFO (QuartzInitializerListener.java:199) - Scheduler has been started...
2020-05-07 18:15:40,280 restartedMain INFO (QuartzInitializerListener.java:217) - Storing the Quartz Scheduler Factory in the servlet context at key: org.quartz.impl.StdSchedulerFactory.KEY
2020-05-07 18:15:40,303 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 18:15:40,549 restartedMain INFO (DruidDataSource.java:994) - {dataSource-1} inited
2020-05-07 18:15:41,667 restartedMain WARN (AbstractApplicationContext.java:558) - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'scheduler'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
2020-05-07 18:15:41,668 restartedMain INFO (DruidDataSource.java:2029) - {dataSource-1} closing ...
2020-05-07 18:15:41,672 restartedMain INFO (DruidDataSource.java:2101) - {dataSource-1} closed
2020-05-07 18:15:41,676 restartedMain INFO (DirectJDKLog.java:173) - Stopping service [Tomcat]
2020-05-07 18:15:41,677 restartedMain INFO (QuartzScheduler.java:666) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutting down.
2020-05-07 18:15:41,677 restartedMain INFO (QuartzScheduler.java:585) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED paused.
2020-05-07 18:15:41,678 restartedMain INFO (QuartzScheduler.java:740) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutdown complete.
2020-05-07 18:15:41,678 restartedMain INFO (QuartzInitializerListener.java:264) - Quartz Scheduler successful shutdown.
2020-05-07 18:15:41,682 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-1] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,683 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-2] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,683 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-3] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,683 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-4] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,684 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-5] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,685 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-6] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,685 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-7] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,686 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-8] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,687 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-9] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,687 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-10] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:15:41,703 restartedMain ERROR (SpringApplication.java:826) - Application run failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'scheduler'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:639)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:116)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:397)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1429)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:879)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:141)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.mof.cc.itsm.ItsmAppliction.main(ItsmAppliction.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.devtools.restart.RestartLauncher.run(RestartLauncher.java:49)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:645)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:475)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1338)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1287)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1207)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:636)
	... 24 common frames omitted
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:640)
	... 37 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1803)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:595)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:339)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$633018df.schedulerFactoryBean(<generated>)
	at org.mof.cc.itsm.config.SchedulerConfig.scheduler(SchedulerConfig.java:55)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$633018df.CGLIB$scheduler$0(<generated>)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$633018df$$FastClassBySpringCGLIB$$760e6bc.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:363)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$633018df.scheduler(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	... 38 common frames omitted
Caused by: java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.createScheduler(SchedulerFactoryBean.java:679)
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.prepareScheduler(SchedulerFactoryBean.java:614)
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.afterPropertiesSet(SchedulerFactoryBean.java:502)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1862)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1799)
	... 57 common frames omitted
2020-05-07 18:17:04,825 restartedMain INFO (StartupInfoLogger.java:55) - Starting ItsmAppliction on DESKTOP-N1ER5LJ with PID 21628 (C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm\target\classes started by CCDC in C:\Users\CCDC\Documents\workspace\sts\maven.1588775637942\itsm)
2020-05-07 18:17:04,828 restartedMain DEBUG (StartupInfoLogger.java:56) - Running with Spring Boot v2.2.0.RELEASE, Spring v5.2.0.RELEASE
2020-05-07 18:17:04,829 restartedMain INFO (SpringApplication.java:655) - The following profiles are active: dev-druid,dev-redis
2020-05-07 18:17:06,233 restartedMain INFO (DirectJDKLog.java:173) - Initializing ProtocolHandler ["http-nio-8080"]
2020-05-07 18:17:06,233 restartedMain INFO (DirectJDKLog.java:173) - Starting service [Tomcat]
2020-05-07 18:17:06,234 restartedMain INFO (DirectJDKLog.java:173) - Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-05-07 18:17:06,319 restartedMain INFO (DirectJDKLog.java:173) - Initializing Spring embedded WebApplicationContext
2020-05-07 18:17:06,444 restartedMain INFO (QuartzInitializerListener.java:147) - Quartz Initializer Servlet loaded, initializing Scheduler...
2020-05-07 18:17:06,466 restartedMain INFO (StdSchedulerFactory.java:1208) - Using default implementation for ThreadExecutor
2020-05-07 18:17:06,468 restartedMain INFO (SimpleThreadPool.java:268) - Job execution threads will use class loader of thread: restartedMain
2020-05-07 18:17:06,476 restartedMain INFO (SchedulerSignalerImpl.java:61) - Initialized Scheduler Signaller of type: class org.quartz.core.SchedulerSignalerImpl
2020-05-07 18:17:06,477 restartedMain INFO (QuartzScheduler.java:229) - Quartz Scheduler v.2.3.1 created.
2020-05-07 18:17:06,478 restartedMain INFO (RAMJobStore.java:155) - RAMJobStore initialized.
2020-05-07 18:17:06,479 restartedMain INFO (QuartzScheduler.java:294) - Scheduler meta-data: Quartz Scheduler (v2.3.1) 'DefaultQuartzScheduler' with instanceId 'NON_CLUSTERED'
  Scheduler class: 'org.quartz.core.QuartzScheduler' - running locally.
  NOT STARTED.
  Currently in standby mode.
  Number of jobs executed: 0
  Using thread pool 'org.quartz.simpl.SimpleThreadPool' - with 10 threads.
  Using job-store 'org.quartz.simpl.RAMJobStore' - which does not support persistence. and is not clustered.

2020-05-07 18:17:06,479 restartedMain INFO (StdSchedulerFactory.java:1362) - Quartz scheduler 'DefaultQuartzScheduler' initialized from default resource file in Quartz package: 'quartz.properties'
2020-05-07 18:17:06,479 restartedMain INFO (StdSchedulerFactory.java:1366) - Quartz scheduler version: 2.3.1
2020-05-07 18:17:06,479 restartedMain INFO (QuartzScheduler.java:547) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED started.
2020-05-07 18:17:06,479 restartedMain INFO (QuartzInitializerListener.java:199) - Scheduler has been started...
2020-05-07 18:17:06,480 restartedMain INFO (QuartzInitializerListener.java:217) - Storing the Quartz Scheduler Factory in the servlet context at key: org.quartz.impl.StdSchedulerFactory.KEY
2020-05-07 18:17:06,502 restartedMain INFO (DruidDataSourceAutoConfigure.java:56) - Init DruidDataSource
2020-05-07 18:17:06,743 restartedMain INFO (DruidDataSource.java:994) - {dataSource-1} inited
2020-05-07 18:17:07,736 restartedMain WARN (AbstractApplicationContext.java:558) - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'scheduler'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
2020-05-07 18:17:07,738 restartedMain INFO (DruidDataSource.java:2029) - {dataSource-1} closing ...
2020-05-07 18:17:07,743 restartedMain INFO (DruidDataSource.java:2101) - {dataSource-1} closed
2020-05-07 18:17:07,745 restartedMain INFO (DirectJDKLog.java:173) - Stopping service [Tomcat]
2020-05-07 18:17:07,746 restartedMain INFO (QuartzScheduler.java:666) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutting down.
2020-05-07 18:17:07,747 restartedMain INFO (QuartzScheduler.java:585) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED paused.
2020-05-07 18:17:07,747 restartedMain INFO (QuartzScheduler.java:740) - Scheduler DefaultQuartzScheduler_$_NON_CLUSTERED shutdown complete.
2020-05-07 18:17:07,747 restartedMain INFO (QuartzInitializerListener.java:264) - Quartz Scheduler successful shutdown.
2020-05-07 18:17:07,749 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-1] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,749 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-2] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,750 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-3] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,750 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-4] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,751 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-5] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,751 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-6] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,752 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-7] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,752 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-8] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,753 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-9] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,754 restartedMain WARN (DirectJDKLog.java:173) - The web application [itsm] appears to have started a thread named [DefaultQuartzScheduler_Worker-10] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.lang.Object.wait(Native Method)
 org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:568)
2020-05-07 18:17:07,767 restartedMain ERROR (SpringApplication.java:826) - Application run failed
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'jobController': Unsatisfied dependency expressed through field 'scheduler'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:639)
	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:116)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessProperties(AutowiredAnnotationBeanPostProcessor.java:397)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1429)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:879)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:878)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:550)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:141)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:747)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at org.mof.cc.itsm.ItsmAppliction.main(ItsmAppliction.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.devtools.restart.RestartLauncher.run(RestartLauncher.java:49)
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'Scheduler' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:645)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:475)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1338)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:557)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:276)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1287)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1207)
	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredFieldElement.inject(AutowiredAnnotationBeanPostProcessor.java:636)
	... 24 common frames omitted
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.quartz.Scheduler]: Factory method 'scheduler' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:185)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiate(ConstructorResolver.java:640)
	... 37 common frames omitted
Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'SchedulerFactory' defined in class path resource [org/mof/cc/itsm/config/SchedulerConfig.class]: Invocation of init method failed; nested exception is java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1803)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:595)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:517)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:323)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:222)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:321)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:339)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$b61e0a04.schedulerFactoryBean(<generated>)
	at org.mof.cc.itsm.config.SchedulerConfig.scheduler(SchedulerConfig.java:55)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$b61e0a04.CGLIB$scheduler$0(<generated>)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$b61e0a04$$FastClassBySpringCGLIB$$80489a56.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invokeSuper(MethodProxy.java:244)
	at org.springframework.context.annotation.ConfigurationClassEnhancer$BeanMethodInterceptor.intercept(ConfigurationClassEnhancer.java:363)
	at org.mof.cc.itsm.config.SchedulerConfig$$EnhancerBySpringCGLIB$$b61e0a04.scheduler(<generated>)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:154)
	... 38 common frames omitted
Caused by: java.lang.IllegalStateException: Active Scheduler of name 'DefaultQuartzScheduler' already registered in Quartz SchedulerRepository. Cannot create a new Spring-managed Scheduler of the same name!
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.createScheduler(SchedulerFactoryBean.java:679)
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.prepareScheduler(SchedulerFactoryBean.java:614)
	at org.springframework.scheduling.quartz.SchedulerFactoryBean.afterPropertiesSet(SchedulerFactoryBean.java:502)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1862)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1799)
	... 57 common frames omitted
